tests/shadow_price.rs

```diff
@@ -63,12 +70,15 @@ mod tests {
             .solve()
             .expect("Library test");
 
-        assert_eq!(solution.eval(&objective) as u32, 4100);
-        assert_eq!(solution.value(n_chairs) as u32, 29);
-        assert_eq!(solution.value(n_tables) as u32, 40);
-        assert_eq!(
-            solution.get_dual_values(),
-            vec![-15.000000000000004, -4.999999999999992]
+        assert_float_eq!(4100.0, solution.eval(&objective), abs <= 1e-10);
+        assert_float_eq!(29.999999999999996, solution.value(n_chairs), abs <= 1e-10);
+        assert_float_eq!(40.00000000000001, solution.value(n_tables), abs <= 1e-10);
```

Comment:
This should probably be 30.0 and 40.0 instead. You can reduce the abs if necessary